Kryonite Labs website MVP (#1)
kryonitelabs/docs: blog/2021-09-16-what-is-kryonite-labs.md

---
slug: what-is-kryonite-labs
title: What is Kryonite Labs?
authors: [itskev, lusu007]
tags: [kryocloud, opensource]
---

## ❗ Our intent

Over the past few years, we have gained a lot of experience in the Minecraft development scene. One of the big problems we have constantly noticed is that many networks and developers are still working with old technology stacks. Unfortunately, many projects are not maintained or are no longer developed at all. With [Kryonite Labs](https://github.com/kryonitelabs/), we want to create a space for new projects with new possibilities for Minecraft.

**We want to offer our users the most modern and best user experience.**

## 🚧 Our projects

### 🌨️ Kryocloud

> The main project of [Kryonite Labs](https://github.com/kryonitelabs/) is our [Kryocloud](./docs/kryocloud/concept). Actually, we wanted to remove the word "cloud" from the name, as this software is not a cloud at all, but an orchestrator. However, the name "cloud" for that kind of software has become widespread in the scene... 🤷‍♂️

Reviewer: ```suggestion
The main project of [Kryonite Labs](https://github.com/kryonitelabs/) is [Kryocloud](./docs/kryocloud/concept). We didn't want to include the word "cloud" in the project's name, as this software is not a cloud, but an orchestrator. However, the name "cloud" for that kind of software has become widespread in the scene... 🤷‍♂️
```
